lib/location/Kconfig

```diff
@@ -126,39 +122,57 @@ config LOCATION_METHOD_WIFI_REST_RECV_BUF_SIZE
 	  Size of the buffer used to store the response from the Wi-Fi positioning
 	  service.
 
-if LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD
+endif # LOCATION_METHOD_WIFI
+
+if LOCATION_METHOD_CELLULAR || LOCATION_METHOD_WIFI
 
-config LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD_HOSTNAME
-	string "nRF Cloud REST API hostname"
-	default NRF_CLOUD_REST_HOST_NAME
+config LOCATION_SERVICE_HERE
+	bool "HERE location service"
 	help
-	  Server hostname to use when connecting to the nRF Cloud location service.
+	  Use HERE location service.
 
-endif # LOCATION_METHOD_WIFI_SERVICE_NRF_CLOUD
+config LOCATION_SERVICE_NRF_CLOUD
+	bool "nRF Cloud location service"
+	default y if !LOCATION_METHOD_CELLULAR_EXTERNAL || !LOCATION_METHOD_WIFI_EXTERNAL
```

Comment:
For cellular, this condition was not here but for WiFi it was there earlier also so this is a small change for cellular side.